araproc/analysis/standard_reco.py

```diff
@@ -701,3 +704,135 @@ def min_frac_corr_depth_multiple(self, *maps, fraction=0.6, z_thresh=0):
             raise RuntimeError("No maps meet the fractional correlation threshold within the z_thresh.")
 
         return min_depth, min_depth_index
+
+    def pairing(channels):
+
+        """
+        Makes channel pairs.
+    
+        Parameters
+        ----------
+        channels: list
+           List of given channels.
+
+        Returns
+        -------
+        pairs: list
+           List of all possible pair combinations of given channels.
+        """
+
+        pairs = list(itertools.combinations(channels, 2))
+
+        return pairs
+
+    def get_corr(y1, y2):
```

Comment:
We can drop this function. You'll use `__get_correlation_function(ch1, ch2, event_num)` to get the correlation TGraph.